.circleci/run_xpu_tests.py

```diff
@@ -0,0 +1,47 @@
+import argparse
```

Comment:
do we want to see this script here? not in private infra?

Reply:
Not a problem to move it to the private repo, but I placed it here because it potentially can be used in public too in the future. And it is connected to the content of other files in this folder

Reply:
I agree, ok